### ISSUE TITLE
Why is Lisa’s training loss always 0?

### ISSUE THREAD

**Issue by orderer0001** [OptimalScale/LMFlow#891]
Why is Lisa’s training loss always 0?

**Comment by research4pan** [comment]
Thanks for your interest in LMFlow! That's strange, we are wondering if you could share more details of the training statistics and fine-tuning options, so we could help check the problem for you?
